internal/cmd/asyncapi/command_export.go

```diff
@@ -218,22 +214,28 @@ func (c *command) getAccountDetails(flags *flags) (*accountDetails, error) {
 	if err := c.getClusterDetails(details, flags); err != nil {
 		return nil, err
 	}
-	if err := c.getSchemaRegistry(details, flags); err != nil {
+
+	client, err := c.GetSchemaRegistryClient()
```

Comment:
```suggestion
	details.srClient, err := c.GetSchemaRegistryClient()
```

Reply:
This won't work, since you can only use `:=` with variables that don't exist yet

Reply:
can we replace `:=` with `=`, since `err` is already defined above and we know `err == nil` up until this point?